pkgdb2/api/extras.py

```diff
@@ -585,3 +586,29 @@ def api_monitored():
             '\n'.join(output),
             content_type="text/plain;charset=UTF-8"
         )
+
+
+@API.route('/dead/package/<pkg_name>/<clt_name>')
+def api_dead_package(pkg_name, clt_name):
+    '''
+    Returned the content of the of dead.package file
+    -----------------------
+    Retired packages should have in their git a ``dead.package`` file
+    containing the explanation as why the package was retired.
+    This method calls cgit to return that explanation.
+
+    ::
+
+        /api/dead/package/acheck/master
+
+    '''
+    req = requests.get(
```

Comment:
Ok I tried to do this, but since this API is designed to be called via jquery/ajax, the 302 triggers the cross-site security which also prevents us from calling cgit directly.

If you have an idea to call cgit directly (or redirect to it and having jquery happy about it), feel free :-)